.gitignore

```diff
@@ -1 +1,4 @@
 .DS_Store
+node_modules
+notes/node_modules
```

Comment:
should be enough to just ignore `node_modules` in the Git project's parent level .gitignore file. The Git project covers all the subfolders and the "ignores" are applied recursively to subfolders. If, however, you are ignoring something on the parent level and want it included in a subfolder you could use the exclamation point to make sure it is _**not**_ ignored, e.g. `!notes/node_modules` would keep the nod mods in the notes folder from being ignored (not that there's any good reason to push nod mods up to GH) 😛